assets/script.js

```diff
@@ -194,40 +209,63 @@ fetch(searchNinjaUrl,
     console.log("search input did not have output. try something else");
     return;  //ends function early for bad search input.
   }
-  console.log(data); 
-  sportResult = data[0];
-  sportCalories = sportResult.calories_per_hour;
-  console.log(sportResult);
-  console.log(sportResult.name);
-  console.log(sportResult.calories_per_hour);
-  console.log(sportCalories);
+  sportInfoCurrent.push(data)
+  // console.log(data); 
+  // sportResult = data[0];
+  // sportCalories = sportResult.calories_per_hour;
+  // console.log(sportResult);
+  // console.log(sportResult.name);
+  // console.log(sportResult.calories_per_hour);
+  // console.log(sportCalories);
 })
 .catch(function (error) {
   console.error(error);
   notFound.textContent = "searchNinjaUrl_error";
 });
+console.log(i)
+}
+if (i >= sportSet.length){
 return;
 }
+}
 
 async function fetchActivities(calories) {
   const sportData = await fetch(``);
 }
 
+
+//------------------->compute-------------------------------
+
+//get duration of sport in minutes to match menu calories
+function computeDuration() {
+  for (var i = 0; i < saveList.length; i++) { 
+    var sportCalories = sportInfoCurrent[i][0].calories_per_hour
+  if (sportCalories == ""){
+    console.log("coumputeDuration function errored");
+    return;
+  }
+  sportDuration = sampleMenuCalories / sportCalories;
+  console.log(sportDuration + "hours")
+  var sportDurationMin = sportDuration * 60;
+  console.log(sportDurationMin.toFixed() + "minutes");
+  return;
+}
+}
+
 //------------------------>set------------------------------
 
 //below are functions to load, save, display, and use locally stored cuisine element
-var saveList = ""; //initial blank savelist at load. Array. store cuisine input.
 
 //load local storage
-function loadSaved() {
-  var saved = JSON.parse(localStorage.getItem("saved"));
-  console.log(saved);
-  if (saved !== null) {
-    saveList = saved;
-  }}
+// function loadSaved() {
+//   var saved = JSON.parse(localStorage.getItem("saved"));
+//   console.log(saved);
+//   if (saved !== null) {
+//     saveList = saved;
+//   }}
 
 //save input value
-function storeSave() {
+function storeSaveSport() {
   saveList.push(cuisineInputEl.value);  //currently set to cuisine input. change if needed
```

Comment:
i think you meant to save your  sportInfoCurrent to localStorage? not the option input, right?

Reply:
Sorry. This is just the sample code I put in earlier. Currently is should do nothing. I plan to refine it tonight so it will function as set local storage for sport info, to be used by second page.